core/src/interfaces.rs

```diff
@@ -442,14 +442,84 @@ impl TryFrom<ProofRequestOpt> for ProofRequest {
     }
 }
 
-#[serde_as]
-#[derive(Clone, Debug, Serialize, Deserialize)]
+#[derive(Default, Clone, Serialize, Deserialize, Debug, ToSchema)]
+#[serde(default)]
 /// A request for proof aggregation of multiple proofs.
 pub struct AggregationRequest {
-    /// All the proofs to verify
-    pub proofs: Vec<Proof>,
+    /// The block numbers and l1 inclusion block numbers for the blocks to aggregate proofs for.
+    pub block_numbers: Vec<(u64, u64)>,
+    /// The network to generate the proof for.
+    pub network: Option<String>,
+    /// The L1 network to generate the proof for.
+    pub l1_network: Option<String>,
+    // Graffiti.
+    pub graffiti: Option<String>,
+    /// The protocol instance data.
+    pub prover: Option<String>,
     /// The proof type.
-    pub proof_type: ProofType,
-    /// Additional prover params.
-    pub prover_args: HashMap<String, Value>,
+    pub proof_type: Option<String>,
+    /// Blob proof type.
+    pub blob_proof_type: Option<String>,
```

Comment:
So that would mean that we should have a request that has a similar param to the following:
```json
{
  ...
  "block_numbers": [
    [123, 121, null],
    [124, null, null],
    [125, 122, "kzg"],
    [126, null, "kzg"],
  ]
  ...
}
```

Where the tuple contains a block number, a optional inclusion and a optional proof type.

Reply:
No need, the blob proof type is unified in 1 proof system, so no problem in aggregation.